avidemux/qt4/i18n/avidemux_da.ts

```diff
@@ -6363,16 +5720,6 @@ Jeg vil antage at den har og prøve at fortsætte</translation>
         <source>Adjust hue and saturation.</source>
         <translation>Justér farvetone og mætning.</translation>
     </message>
-    <message>
-        <location filename="../../../avidemux_plugins/ADM_videoFilters6/hue/qt4/DIA_flyHue.cpp" line="+61"/>
```

Comment:
The following strings have been made translatable 5 months ago, your source tree must be grossly out of date.